app/helpers/thredded/icon_helper.rb

```diff
@@ -2,6 +2,17 @@
 
 module Thredded
   module IconHelper
+    def svg_icon(filename, **args)
```

Comment:
`shared_inline_svg`?

Reply:
Yes good. I’ll do that then merge this into the other branch

Reply:
Yes good. I’ll do that then merge this into the other branch